featurebyte/service/feature.py

```diff
@@ -176,15 +175,6 @@ async def create_document(self, data: FeatureServiceCreate) -> FeatureModel:
             )
             assert insert_id == document.id
 
-            # initialize offline store info
-            if FeastIntegrationSettings().FEATUREBYTE_FEAST_INTEGRATION_ENABLED:
-                store_info = await self.offline_store_info_initialization_service.initialize_offline_store_info(
-                    feature=document
-                )
-                await self.update_offline_store_info(
-                    document_id=document.id, store_info=store_info.dict(by_alias=True)
-                )
```

Comment:
Move this to deployment flow so that this attribute is generated only the feature is online-enabled.